extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTask.java

```diff
@@ -1008,7 +1008,7 @@ private void possiblyResetOffsetsOrWait(
         final TopicPartition topicPartition = outOfRangePartition.getKey();
         final long nextOffset = outOfRangePartition.getValue();
         // seek to the beginning to get the least available offset
-        consumer.seekToBeginning(topicPartition);
+        consumer.seekToBeginning(Lists.newArrayList(topicPartition));
```

Comment:
Also in  #4135 you mentioned prefer `Collections.emptyList()`  over `Arrays.asList()`  then shall i also replace `Lists.newArrayList()` with `Collections.emptyList()` ?